HCCGo/app/js/jobSubmissionCtrl.js

```diff
@@ -55,6 +60,14 @@ jobSubmissionModule.controller('jobSubmissionCtrl', ['$scope', '$log', '$timeout
     editor.setValue($scope.job.commands);
   }
 
+  $scope.locationPath = function() {
+    if($scope.job.location.search(/^\w*(\.slurm|\.err|\.out)$/)!=-1 |
+     $scope.job.error.search(/^\w*(\.slurm|\.err|\.out)$/)!=-1 | 
+     $scope.job.output.search(/^\w*(\.slurm|\.err|\.out)$/)!=-1){
+     $scope.jobscript.changePath = true;
```

Comment:
So, if any of the location, error, output has a wrong page, then all of those spans show up?